api/s2n.h

```diff
@@ -2162,9 +2164,13 @@ extern int s2n_connection_set_session(struct s2n_connection *conn, const uint8_t
 /**
  * Serializes the session state from connection and copies into the `session` buffer and returns the number of copied bytes
  *
- * The output of this function depends on whether session ids or session tickets are being used for resumption.
+ * The output of this function depends on whether session ids or session tickets are being used for resumption. If the first 
+ * byte in **session** is 1, then the next 2 bytes will contain the session ticket length, followed by session ticket and
+ * session state. In versions TLS1.3 and greater, (which allows multiple session tickets), the most recent session ticket
+ * received will be used. Note that the size of the session tickets varies.
+ * If the first byte in **session** is 0, then the next byte will contain session id length, followed by session id and session state.
```

Comment:
Should we really be going into the byte layout of the output? Do we guarantee that won't change?

(Did we already change it for TLS1.3?)

Reply:
Yeah....I really don't want to go into this level of detail. I just didn't want to delete this section from the Usage Guide without attempting some sort of equivalence. I am honestly not sure if this information is necessary. 

It didn't change for TLS1.3, it's just describing the wrapper around the ticket.

Reply:
I think I'm just going to delete it.